io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,136 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --filesystem=~/.var/app:ro
+- --filesystem=home
+- --filesystem=/run/media:ro
+- --filesystem=/media:ro
+
+cleanup:
+- /include
+- /lib/*.a
+- /lib/*.la
+- /lib/pkgconfig
+- /lib/cmake
+- /lib/debug
```

Comment:
```suggestion
```

I missed that one. that's the build failure.

Reply:
Why that causes build error?

Reply:
because removing the debug symbols is bad. they get split out in a separate package anyway.